packages/core/src/generator/index.ts

```diff
@@ -280,15 +280,15 @@ export class UnoGenerator<Theme extends object = object> {
       }
 
       const layerMark = minify ? '' : `/* layer: ${layer} */${nl}`
-      return layerCache[layer] = css ? layerMark + css : ''
+      return layerCache[layer] = sortImport(css ? layerMark + css : '')
     }
 
     const getLayers = (includes = layers, excludes?: string[]) => {
-      return includes
+      return sortImport(includes
```

Comment:
But honestly, I don't think this is the best way to do so, as this would affect every generation for a relevantly rare use case.

I think it might be better to have a separated layer for the `@import` to be generated to